applications/MedApplication/custom_io/med_model_part_io.cpp

```diff
@@ -67,10 +67,26 @@ void CheckMEDErrorCode(const int ierr, const std::string& MEDCallName)
     KRATOS_ERROR_IF(ierr < 0) << MEDCallName << " failed with error code " << ierr << "." << std::endl;
 }
 
+bool IsNotPaddingCharacter(std::string::value_type character) noexcept
+{
+    return !(std::isspace(character) || character == '\0');
+}
+
 // The names in the MED-file often have trailing null-chars, which need to be removed
 // this can otherwise make debugging very tricky
-void RemoveTrailingNullChars(std::string& rInput)
+void RemovePadding(std::string& rInput)
 {
+    // Trime left
+    rInput.erase(rInput.begin(),
+                 std::find_if(rInput.begin(),
+                              rInput.end(),
+                              IsNotPaddingCharacter));
+
+    // Trim right
+    rInput.erase(std::find_if(rInput.rbegin(),
+                              rInput.rend(),
+                              IsNotPaddingCharacter).base(),
+                 rInput.end());
     rInput.erase(std::find(rInput.begin(), rInput.end(), '\0'), rInput.end());
 }
```

Comment:
hmm this looks now like a pretty good addition to the [StringUtilities](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/utilities/string_utilities.h)

Could you please add it? I can review fast ;)

Not a blocker, we can also merge this one first.

Ideally this should have tests, but would not fit the current structure of the code, but once it is in the StringUtils

Reply:
see #12225